Validators.js

```diff
@@ -159,4 +159,17 @@ function areEventParametersValid(functionName, eventName, segmentation, eventCou
     return true;
 }
 
-export { validateUserDataValue as UserDataValue, validateString as String, validateParseInt as ParseInt, validateValidUserData as ValidUserData, validateUserDataType as UserDataType, areEventParametersValid };
+function isValidPrimitiveOrArray(value) {
+    return (
+        typeof value === 'string' ||
+        typeof value === 'number' ||
+        typeof value === 'boolean' ||
+        Array.isArray(value) && value.every(item => (
```

Comment:
cant we leave the array's internal type checks to underlying sdk? its not behaving as expected?